examples/plot_interpolate_bad_channels.py

```diff
@@ -1,10 +1,15 @@
 """
-================================================
-Interpolate bad channels using spherical splines
-================================================
+=============================================
+Interpolate bad channels for MEG/EEG channels
+=============================================
 
-This example shows how to interpolate bad EEG channels using spherical
-splines as described in [1].
+This example shows how to interpolate bad MEG/EEG channels
+
+    - Using spherical splines as described in [1] for EEG data.
```

Comment:
`[1]_`, and then use `.. [1]` below, see e.g. https://github.com/mne-tools/mne-python/blob/master/mne/filter.py#L1421